picasso/src/main/java/com/squareup/picasso3/Picasso.java

```diff
@@ -745,6 +748,17 @@ public Builder listener(@NonNull Listener listener) {
       return this;
     }
 
+    /** Add an decoder that can decode custom image formats. */
+    @NonNull
+    public Builder addImageDecoder(@NonNull ImageDecoder imageDecoder) {
```

Comment:
Allows system wide addition of new ImageDecoders, so you could add your own MP4->Drawable decoder, for example.